nexus/reconfigurator/planning/src/planner.rs

```diff
@@ -901,6 +912,63 @@ impl<'a> Planner<'a> {
         Ok(())
     }
 
+    /// Update at most one MGS-managed device (SP, RoT, etc.), if any are out of
+    /// date.
+    fn do_plan_mgs_updates(&mut self) -> Result<UpdateStepResult, Error> {
+        // Determine which baseboards we will consider updating.
+        //
+        // Sleds may be present but not adopted as part of the control plane.
+        // In deployed systems, this would probably only happen if a sled was
+        // about to be added.  In dev/test environments, it's common to leave
+        // some number of sleds out of the control plane for various reasons.
+        // Inventory will still report them, but we don't want to touch them.
+        //
+        // For better or worse, switches and PSCs do not have the same idea of
+        // being adopted into the control plane.  If they're present, they're
+        // part of the system, and we will update them.
```

Comment:
Should we have an issue about this? I don't think trust quorum interacts with non-sled components at all, but I think in the fullness of time we want some kind of auth on the management network, which presumably involves the control plane being aware of PSCs/switches and knowing whether or not it's okay to talk to them?

Reply:
We certainly could.  I'm not sure exactly what I'd file at this point.  Something like: "eventually we will have the business requirement to lock down this network, and when we do, we'll have to better manage the lifecycle of these components".  Or "lifecycle of switches and PSCs could be controlled, like sleds". 

I know @bnaecker has been thinking about this a bit in the context of multi-rack.